src/core/message.ts

```diff
@@ -28,6 +28,17 @@ export enum DwnMethodName {
   Delete = 'Delete'
 }
 
+export enum DwnMessageType {
```

Comment:
Adding this since I've noticed that the `web5` package has a notion of dwn message type. i've been wanting to add this for a while, and I intend to do more with it in a tangentially related follow up PR.

In this PR, I'm only adding the DWN messages that are fully supported and ready for use. If I'm missing any or if reviewers would prefer I add everything all at once, i can do that too.

Reply:
Can you just briefly describe where you'll be using it more to demonstrate the need?

Not super concerned either way but right now I am mainly seeing that one parameter is passed instead of two, to which I am not convinced it is worthwhile because of the overhead (albeit minor) of context knowledge of new type and maintenance of it.

Reply:
I was planning to use it to organize message types, reply types, and handlers, sorta similar to what we did in `dwn-types` in #384. After messing around with it for half an hour, I've convinced myself that it's not as useful for that purpose as I thought 🙈.